### ISSUE TITLE
Does ARM now supported by ObjWriter?

### ISSUE THREAD

**Issue by kant2002** [dotnet/runtimelab#1526]
I'm interested technically, given that ARM is building now, that's mean that ObjWriter published to Nuget? And technically ILC can build ARM without specific problems.

**Comment by kant2002** [comment]
I probably should formulate my questions more clearly.

Will `dotnet publish -r linux-arm /p:DisableUnsupportedError=true` works without local build If I change 
https://github.com/dotnet/runtimelab/blob/8ff497b68cc4880ca404958be92c0a1dc2b5c52e/src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj#L34

Or there some other work needed?

**Comment by jkotas** [comment]
Yes, it should. Does it work for you?

**Comment by kant2002** [comment]
No, right now it does not work if I use what's in Nuget 
```
pi@raspberrypi:~/runtimelab/samples/HelloWorld $ ../../.dotnet/dotnet publish -c Release -r linux-arm --packages pkg /p:DisableUnsupportedError=true
Microsoft (R) Build Engine version 17.0.0-preview-21427-02+414393fc1 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  Restored /home/pi/runtimelab/samples/HelloWorld/HelloWorld.csproj (in 5.02 min).
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
/home/pi/runtimelab/.dotnet/sdk/6.0.100-rc.1.21430.12/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(1105,5): warning NETSDK1179: One of '--self-contained' or '--no-self-contained' options are required when '--runtime' is used. [/home/pi/runtimelab/samples/HelloWorld/HelloWorld.csproj]
  HelloWorld -> /home/pi/runtimelab/samples/HelloWorld/bin/Release/net6.0/linux-arm/HelloWorld.dll
/home/pi/runtimelab/samples/HelloWorld/pkg/microsoft.dotnet.ilcompiler/7.0.0-alpha.1.21457.2/build/Microsoft.NETCore.Native.Publish.targets(75,5): error : The PrivateSdkAssemblies ItemGroup is required for _ComputeAssembliesToCompileToNative [/home/pi/runtimelab/samples/HelloWorld/HelloWorld.csproj]
pi@raspberrypi:~/runtimelab/samples/HelloWorld $ ls pkg/
microsoft.aspnetcore.app.runtime.linux-arm  microsoft.dotnet.ilcompiler  microsoft.netcore.app.runtime.linux-arm
```

I see missing `runtime.linux-arm.microsoft.dotnet.ilcompiler`.

Will check with local build in near future.

**Comment by jkotas** [comment]
You may need to fix src\installer\pkg\projects\Microsoft.DotNet.ILCompiler\ILCompilerRIDs.props

**Comment by kant2002** [comment]
Works now. In a sense that it produce binary. Obviously I have to fix #1517 to call that "works"
